mailbox/store/src/main/java/org/apache/james/mailbox/store/StoreMailboxManager.java

```diff
@@ -634,7 +634,7 @@ private void doRenameMailbox(Mailbox mailbox, MailboxPath newMailboxPath, Mailbo
         Flux<Mailbox> baseMailboxes = mailboxMapper
             .findMailboxWithPathLike(toSingleUserQuery(mailboxQuery, session));
         Flux<Mailbox> delegatedMailboxes = getDelegatedMailboxes(mailboxMapper, mailboxQuery, right, session);
-        return Flux.concat(baseMailboxes, delegatedMailboxes)
+        return Flux.merge(baseMailboxes, delegatedMailboxes)
```

Comment:
Why did you move from concat to merge ?

Reply:
@chibenwa switched it to `merge`, I used `concat` initially